dump/mapping/bufr_ssmis.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python3
+import os
+import numpy as np
+import numpy.ma as ma
+import pytz
+from pysolar.solar import get_altitude, get_azimuth
+from datetime import datetime, timezone
+from multiprocessing import Pool, cpu_count
+
+import bufr
+from bufr.obs_builder import ObsBuilder, add_main_functions
+
+
+def map_path(map_file_name):
+    script_dir = os.path.dirname(os.path.abspath(__file__))
+    return os.path.join(script_dir, map_file_name)
+
+
+def get_default_nprocs(default=8):
```

Comment:
I wonder if this should be placed in the backend like we did with some of the other commonly used functions, mostly because it seems like something a number of other scripts will need to use.

Reply:
Let's check with @rmclaren about this.